server/service/microsoft_mdm.go

```diff
@@ -10,12 +10,12 @@ import (
 	"errors"
 	"fmt"
 	"html"
-	"html/template"
 	"io"
 	"net/http"
 	"net/url"
 	"strconv"
 	"strings"
+	"text/template"
```

Comment:
Did you mean to switch back to text/template instead of html/template?

Reply:
I ended up reverting the changes to use "text/template" instead of "html/template" because "html/template" was escaping an URL on the HTML content being returned, which caused the MDM flow to not work

Reply:
Gotcha! 

@roperzh, any ideas on ways we could improve around this? I don't think we need to hold up merging this PR, but it would be great to establish some shared patterns and practices.

Reply:
since this is not a released feature yet, my vote is to create a ticket to deal with the potential code injection appropriately and give more thought (+1 to what Sarah is saying about establishing patterns here)

sorry the `html/template` solution didn't work!

Reply:
Good point. @gillespi314 @roperzh just created https://github.com/fleetdm/fleet/issues/12904 to track this. @gillespi314 should I assign this to you?